.claude/workflows/security-hardening.yaml

```
@@ -0,0 +1,200 @@
+name: "Security Hardening Workflow"
+description: "Comprehensive security hardening with compliance automation"
+version: "1.0"
+
+workflows:
+  security_baseline:
+    name: "Security Baseline Assessment"
+    description: "Establish security baseline for new projects"
+    triggers:
+      - pattern: ["security baseline", "initial security", "new project security"]
+      - context: ["project initialization", "security assessment"]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)